### ISSUE TITLE
CVENotifier is posting vulns that do not seem to match my keywords list

### ISSUE THREAD

**Issue by g33k247** [dark-warlord14/CVENotifier#3]
I am getting vulns for wordpress even though I do not have wordpress or elementor in my keywords list.

**Comment by dark-warlord14** [comment]
Hi @g33k247,

Thank you for reaching out. I will need more information to help with the issue.

Please make sure that you are running the latest version of CVENotifier.
Make sure you are logging the output by running the tool using the following command

```
CVENotifier -config config.yaml 2>&1 | tee CVENotifier.log
```

Now, could you please provide me with the following details:

1. Content of the log file.
2. Keywords from the config file. (please remove the slack webhook before sharing the content of the config file)

Best,
@dark-warlord14

**Comment by g33k247** [comment]
Hell @dark-warlord14. Sorry for my late reply! Here is the info you asked about:

```
2024/05/26 19:18:27 Connected to DB
2024/05/26 19:18:27 vulndb table created successfully
2024/05/26 19:18:27 Matched Keyword: Tor
2024/05/26 19:18:27 Title: CVE-2024-5397 | itsourcecode Online Student Enrollment System 1.0 instructorSubjects.php instructorId sql injection
2024/05/26 19:18:27 Link: https://vuldb.com/?id.266311
2024/05/26 19:18:27 Published Date: Sun, 26 May 2024 10:55:28 +0200
2024/05/26 19:18:27 Categories: Vendor: itsourcecode,Product: Online Student Enrollment System,Risk: critical,Local: No,Remote: Yes,Exploit: Yes,Countermeasures: Unknown
2024/05/26 19:18:27 Insert Completed
2024/05/26 19:18:27 Message sent successfully!
2024/05/26 19:18:27 Matched Keyword: Tor
2024/05/26 19:18:27 Title: CVE-2024-5395 | itsourcecode Online Student Enrollment System 1.0 listofinstructor.php FullName sql injection
2024/05/26 19:18:27 Link: https://vuldb.com/?id.266309
2024/05/26 19:18:27 Published Date: Sun, 26 May 2024 10:55:25 +0200
2024/05/26 19:18:27 Categories: Vendor: itsourcecode,Product: Online Student Enrollment System,Risk: critical,Local: No,Remote: Yes,Exploit: Yes,Countermeasures: Unknown
2024/05/26 19:18:27 Insert Completed
2024/05/26 19:18:27 Message sent successfully!
2024/05/26 19:18:27 Matched Keyword: Tor
2024/05/26 19:18:27 Title: CVE-2024-5379 | JFinalCMS up to 20240111 /admin/template directory cross site scripting (I8VHGR)
2024/05/26 19:18:27 Link: https://vuldb.com/?id.266291
2024/05/26 19:18:27 Published Date: Sat, 25 May 2024 08:35:13 +0200
2024/05/26 19:18:27 Categories: Product: JFinalCMS,Risk: problematic,Local: No,Remote: Yes,Exploit: Yes,Countermeasures: Unknown
2024/05/26 19:18:27 Insert Completed
2024/05/26 19:18:27 Message sent successfully!
2024/05/26 19:18:27 Matched Keyword: Tor
2024/05/26 19:18:27 Title: CVE-2024-5362 | SourceCodester Online Hospital Management System 1.0 departmentDoctor.php deptid sql injection
2024/05/26 19:18:27 Link: https://vuldb.com/?id.266274
2024/05/26 19:18:27 Published Date: Sat, 25 May 2024 08:13:05 +0200
2024/05/26 19:18:27 Categories: Vendor: SourceCodester,Product: Online Hospital Management System,Risk: critical,Local: No,Remote: Yes,Exploit: Yes,Countermeasures: Unknown
2024/05/26 19:18:27 Insert Completed
2024/05/26 19:18:27 Message sent successfully!
2024/05/26 19:18:27 Matched Keyword: Tor
2024/05/26 19:18:27 Title: CVE-2024-29848 | Ivanti Avalanche FileStoreConfig unrestricted upload
2024/05/26 19:18:27 Link: https://vuldb.com/?id.266261
2024/05/26 19:18:27 Published Date: Sat, 25 May 2024 07:27:05 +0200
2024/05/26 19:18:27 Categories: Vendor: Ivanti,Product: Avalanche,Risk: critical,Local: No,Remote: Yes,Exploit: No,Countermeasures: Upgrade
2024/05/26 19:18:27 Insert Completed
2024/05/26 19:18:27 Message sent successfully!
2024/05/26 19:18:27 Matched Keyword: Tor
2024/05/26 19:18:27 Title: CVE-2024-5229 | Primary Addon for Elementor Plugin up to 1.5.5 on WordPress Pricing Table Widget cross site scripting
2024/05/26 19:18:27 Link: https://vuldb.com/?id.266241
2024/05/26 19:18:27 Published Date: Fri, 24 May 2024 20:46:48 +0200
2024/05/26 19:18:27 Categories: Product: Primary Addon for Elementor Plugin,Type: WordPress Plugin,Risk: problematic,Local: No,Remote: Yes,Exploit: No,Countermeasures: Unknown
2024/05/26 19:18:27 Insert Completed
2024/05/26 19:18:27 Message sent successfully!
2024/05/26 19:18:27 Matched Keyword: Tor
2024/05/26 19:18:27 Title: CVE-2024-3552 | Web Directory Free Plugin up to 1.6.9 on WordPress sql injection
2024/05/26 19:18:27 Link: https://vuldb.com/?id.266239
2024/05/26 19:18:27 Published Date: Fri, 24 May 2024 20:46:30 +0200
2024/05/26 19:18:27 Categories: Product: Web Directory Free Plugin,Type: WordPress Plugin,Risk: critical,Local: No,Remote: Yes,Exploit: No,Countermeasures: Unknown
2024/05/26 19:18:27 Insert Completed
2024/05/26 19:18:28 Message sent successfully!
2024/05/26 19:18:28 Matched Keyword: Tor
2024/05/26 19:18:28 Title: CVE-2024-4858 | Testimonial Carousel for Elementor Plugin up to 10.2.0 on WordPress Setting authorization
2024/05/26 19:18:28 Link: https://vuldb.com/?id.266238
2024/05/26 19:18:28 Published Date: Fri, 24 May 2024 20:46:18 +0200
2024/05/26 19:18:28 Categories: Product: Testimonial Carousel for Elementor Plugin,Type: WordPress Plugin,Risk: problematic,Local: No,Remote: Yes,Exploit: No,Countermeasures: Unknown
2024/05/26 19:18:28 Insert Completed
2024/05/26 19:18:28 Message sent successfully!
```

**keywords:**
- Apache HTTP Server
- Apache Tomcat
- Atlassian Confluence
- Cisco ASA
- Cisco NX-OS
- ClamAV
- cURL
- Elasticsearch
- GNU binutils
- Google Chrome
- Kubernetes
- LibreOffice
- Microsoft Edge
- Microsoft Windows
- Mozilla Firefox
- Nagios
- ntp
- OpenSSL
- pfSense
- PuTTY
- Python
- Red Hat Ansible
- Red Hat Enterprise Linux
- Splunk
- SSH
- Suricata
- systemd
- tcpdump
- Tor
- vim
- VMware ESXi
- wireshark

**Comment by dark-warlord14** [comment]
> 2024/05/26 19:18:27 Matched Keyword: **Tor**
> 2024/05/26 19:18:27 Title: CVE-2024-29848 | Ivanti Avalanche FileS**tor**eConfig unrestricted upload
> 2024/05/26 19:18:27 Link: https://vuldb.com/?id.266261
> 2024/05/26 19:18:27 Published Date: Sat, 25 May 2024 07:27:05 +0200
> 2024/05/26 19:18:27 Categories: Vendor: Ivanti,Product: Avalanche,Risk: critical,Local: No,Remote: Yes,Exploit: No,Countermeasures: Upgrade
> 2024/05/26 19:18:27 Insert Completed
> 2024/05/26 19:18:27 Message sent successfully!
> 2024/05/26 19:18:27 Matched Keyword: **Tor**
> 2024/05/26 19:18:27 Title: CVE-2024-5229 | Primary Addon for Elemen**tor** Plugin up to 1.5.5 on WordPress Pricing Table Widget cross site scripting
> 2024/05/26 19:18:27 Link: https://vuldb.com/?id.266241
> 2024/05/26 19:18:27 Published Date: Fri, 24 May 2024 20:46:48 +0200
> 2024/05/26 19:18:27 Categories: Product: Primary Addon for Elementor Plugin,Type: WordPress Plugin,Risk: problematic,Local: No,Remote: Yes,Exploit: No,Countermeasures: Unknown
> 2024/05/26 19:18:27 Insert Completed
> 2024/05/26 19:18:27 Message sent successfully!
> 2024/05/26 19:18:27 Matched Keyword: **Tor**
> 2024/05/26 19:18:27 Title: CVE-2024-3552 | Web Direc**tor**y Free Plugin up to 1.6.9 on WordPress sql injection
> 2024/05/26 19:18:27 Link: https://vuldb.com/?id.266239
> 2024/05/26 19:18:27 Published Date: Fri, 24 May 2024 20:46:30 +0200
> 2024/05/26 19:18:27 Categories: Product: Web Directory Free Plugin,Type: WordPress Plugin,Risk: critical,Local: No,Remote: Yes,Exploit: No,Countermeasures: Unknown
> 2024/05/26 19:18:27 Insert Completed
> 2024/05/26 19:18:28 Message sent successfully!
> 2024/05/26 19:18:28 Matched Keyword: **Tor**
> 2024/05/26 19:18:28 Title: CVE-2024-4858 | Testimonial Carousel for Elemen**tor** Plugin up to 10.2.0 on WordPress Setting authorization
> 2024/05/26 19:18:28 Link: https://vuldb.com/?id.266238
> 2024/05/26 19:18:28 Published Date: Fri, 24 May 2024 20:46:18 +0200
> 2024/05/26 19:18:28 Categories: Product: Testimonial Carousel for Elementor Plugin,Type: WordPress Plugin,Risk: problematic,Local: No,Remote: Yes,Exploit: No,Countermeasures: Unknown
> 2024/05/26 19:18:28 Insert Completed
> 2024/05/26 19:18:28 Message sent successfully!

The **tor** keyword is causing the issue.
Whenever you see too much noise, find out the matched keyword from the log and try to be specific or just remove it.
In this case, you can either try with **tor browser** or could just remove it.

**Comment by dark-warlord14** [comment]
@g33k247 Please check if the above fixes the issue and then we can close this one.

Have a good one!

**Comment by g33k247** [comment]
@dark-warlord14  thanks for your quick response and spot-on advice! I actually looked at the log before sending it over to you and immediately removed tor from my keyword list. So thank you also for reminding me to review logs, logs, logs! I am wondering if it's posible to quote spaces before and after a keyword, like " tor " and if that would achieve a similar result. I'll look into that.  Closing issue.
